CameraOverlay/OrchestratorClass.py

```diff
@@ -0,0 +1,79 @@
+"Orchestrator Script That Controls The Camera System"
```

Comment:
Use triple quotes (`"""`) for documentation comments (ones at the start of files and functions)

Reply:
is it wrong to use '#' for comments

Reply:
Not wrong, but it is convention to use triple quotes for documentation (when you’re describing a function, class, or file). This way automated tools will know it’s documentation. For instance VS Code will put it in their autocomplete